GPU Memory allocated after: 0.0048 GB
Traceback (most recent call last):
  File "/gpfs/home5/scur2687/erwin/experiments/train_shapenet.py", line 146, in <module>
    fit(config, model, optimizer, scheduler, train_loader, valid_loader, test_loader, 110, 160)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/training.py", line 293, in fit
    n_influenced = evaluate_interactions_from_batch(model, next(iter(train_loader)), config)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/training.py", line 138, in evaluate_interactions_from_batch
    x = batch['node_features']
KeyError: 'node_features'
Traceback (most recent call last):
  File "/gpfs/home5/scur2687/erwin/experiments/train_shapenet.py", line 146, in <module>
    fit(config, model, optimizer, scheduler, train_loader, valid_loader, test_loader, 110, 160)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/training.py", line 293, in fit
    n_influenced = evaluate_interactions_from_batch(model, next(iter(train_loader)), config)
  File "/gpfs/home5/scur2687/erwin/experiments/../../erwin/training.py", line 138, in evaluate_interactions_from_batch
    x = batch['node_features']
KeyError: 'node_features'
